Merge Working Groups and Collaboration Spaces (#1339)
openjs-foundation/cross-project-council: collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md

This governance policy describes how a new `Collaboration Space` can be established within the OpenJS Foundation, hereinafter referred to as the "Foundation", via the [Collaboration Space Proposal Process](#ii-collaboration-space-proposal-process).
It describes the [Stages](#stages) a collaboration space may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a space to move from one stage to another.

The Cross Project Council (CPC) may [formally delegate authority](./COLLABORATION_NETWORK.md#authority-delegation) to a `Collaboration Space`, so that the `Collaboration Space` can make autonomous decisions with respect to the delegated scope.

> `Collaboration Spaces` are intended to expand the group of contributors to the work of the OpenJS Foundation's work by allowing people outside the OpenJS member projects to propose/collaborate on areas of importance to the JavaScript ecosystem.

Reviewer: ```suggestion
`Collaboration Spaces` aim to expand the group of contributors to the OpenJS Foundation by enabling individuals outside of OpenJS Foundation projects to propose and collaborate on areas important to the JavaScript ecosystem.
```

suggestion for clarity
